.circleci/config.yml

```diff
@@ -19,19 +20,21 @@ jobs:
             gcloud auth activate-service-account --key-file=${HOME}/gcloud-service-key.json
             gcloud --quiet config set project dev-qlik-core
             gcloud --quiet config set compute/zone $TEST_CLUSTER_ZONE
-      - run:
-          name: Accept EULA
-          command: grep -rl AcceptEULA ./qlik-core | xargs sed -i 's/AcceptEULA=no/AcceptEULA=yes/g'
-      - run:
-          name: Set Disk Name For Test Cluster
-          command: grep -rl my-disk ./nfs-volumes | xargs sed -i "s/my-disk/$DISK_NAME_TEST-$CIRCLE_BUILD_NUM/g"
       - run:
           name: Deploy test cluster
           environment:
             GCLOUD_NUM_NODES: 2
           command: |
-            GCLOUD_ZONE=$TEST_CLUSTER_ZONE K8S_CLUSTER=cci-cluster-$CIRCLE_BUILD_NUM DISK_NAME=$DISK_NAME_TEST-$CIRCLE_BUILD_NUM ./run.sh deploy
-            # Wait for all deployments to rollout
+            curl https://storage.googleapis.com/kubernetes-helm/helm-v2.9.0-linux-amd64.tar.gz | tar xz
```

Comment:
Why 2.9.0?
https://github.com/helm/helm/releases

Reply:
It was a great release, bumped to latest